glad/documentation.py

```diff
@@ -0,0 +1,171 @@
+from glad.parse import SpecificationDocs, CommandDocs, xml_parse
+from glad.util import prefix, memoize, raw_text
+from shutil import rmtree
+import glad.util
+import subprocess
+import re
+
+
+class DocsGL(SpecificationDocs):
+    URL = 'https://github.com/BSVino/docs.gl.git'
+    CACHED = True # Only clones the DocsGL repo once
+    API = 'gl'
+    docs = dict()
+
+    @property
+    @memoize(method=True)
+    def out_dir(self):
+        from pathlib import Path
+        from tempfile import gettempdir
+        if self.CACHED:
+            return Path('.cached') / 'docs.gl'
+        return Path(gettempdir()) / 'docs.gl'
```

Comment:
This should either be a field on the instance or just a temporary variable.

Reply:
Removed and move this logic to `__main__.py`, like specification does